src/Workspaces/CSharp/Portable/Extensions/ExpressionSyntaxExtensions.cs

```diff
@@ -232,6 +232,9 @@ public static bool IsLeftSideOfQualifiedName(this ExpressionSyntax expression)
                 expression.IsParentKind(SyntaxKind.QualifiedName) && ((QualifiedNameSyntax)expression.Parent).Left == expression;
         }
 
+        public static bool IsLeftSideOfExplicitInterfaceSpecifier(this NameSyntax name)
+            => name.IsParentKind(SyntaxKind.ExplicitInterfaceSpecifier);
```

Comment:
Technically correct... but feels wonky to me a bit that you're not actually checking you're the name.  don't have to hcange this if you don't want to.